app/src/main/res/layout/administrator_controls_general_view.xml

```diff
@@ -18,9 +18,9 @@
       style="@style/Heading5ViewStart"
       android:layout_width="match_parent"
       android:background="@color/admin_controls_sub_heading_background_color"
-      android:paddingStart="16dp"
-      android:paddingTop="20dp"
-      android:paddingEnd="16dp"
+      android:paddingStart="@dimen/administrator_controls_general_view_general_text_view_padding_start"
+      android:paddingTop="@dimen/administrator_controls_general_view_general_text_view_padding_top"
+      android:paddingEnd="@dimen/administrator_controls_general_view_general_text_view_padding_end"
```

Comment:
I am not sure if this way of naming dimens is acceptable or not. I think something like `administrator_controls_general_view_sub_heading_text_view_padding_start` and `administrator_controls_general_view_options_text_view_padding_start` (for lower text view) will be better for consistency with the colors.